### ISSUE TITLE
Download button has conflicting attributes

### ISSUE THREAD

**Issue by cjrace** [dfe-analytical-services/shiny-template#97]
The ‘Download data’ link has conflicting attributes applied to it, which might be confusing for assistive technology.

Page 46 of the [DAC report](https://educationgovuk.sharepoint.com/sites/lveesfa00074/Data%20Insights%20and%20Statistics%20Division/Forms/AllItems.aspx?id=%2Fsites%2Flveesfa00074%2FData%20Insights%20and%20Statistics%20Division%2FStatistics%20Services%20Unit%2FUnit%20projects%2FAccessibility%20audit%202023%2D24%2FDAC%20Accessibility%20Audit%20Report%20WCAG%202%2E2%20%2D%20Department%20for%20Education%20%2D%20Explore%20Education%20Statistics%2EDOC%2Epdf&viewid=6a884e94%2D19d7%2D454f%2D8aa1%2D5ca22676ac00&parent=%2Fsites%2Flveesfa00074%2FData%20Insights%20and%20Statistics%20Division%2FStatistics%20Services%20Unit%2FUnit%20projects%2FAccessibility%20audit%202023%2D24) for more details

Currently the <i> element found within the link has the ‘role="presentation"’ attribute applied which should programmatically hide the element from assistive technology, however due to the implementation of the ‘aria-label’ attribute this overrides the attribute, which means it is discoverable to assistive technology. As a result of this the link will have the accessible name of ‘download icon Download data’.

Also, there is an ‘aria-live’ attribute applied which should only be used to inform assistive technologies about regions of a web page that are dynamically updated and should be announced to users in a non-interruptive manner. In its current form this does not provide 
screen reader users with a status message, however the inclusion of this attribute is not suitable for this scenario.

Additional instances of this issue may exist on other pages throughout the website; wherever this issue occurs, they too will need to be resolved.

## Solution
Remove the ‘aria-label’ and the ‘aria-live’ attribute as they are not required. The ‘role="presentation"’ attribute will inform assistive technologies that an element should be treated as purely presentational and should not be announced to the user.
Removing the ‘aria-label’ will provide the link with an accessible name of ‘Download data’ which is expected.

**Comment by cjrace** [comment]
In the development version of shinyGovstyle there is now a download_link() function that gives a standard component for downloading files, we should switch to using this - https://dfe-analytical-services.github.io/shinyGovstyle/reference/download_link.html
